sdk-base/src/main/java/com/mapbox/maps/plugin/delegates/MapProjectionDelegate.kt

```diff
@@ -29,6 +34,10 @@ interface MapProjectionDelegate {
   /**
    * Convert to a camera options from a given list of points, padding, bearing and pitch values.
    *
+   * In order for this method to produce correct results MapView must be already
```

Comment:
nit : the doc in the delegate and mapboxmap seems doesn't exactly match, could use `[MapView]` here.

Reply:
because we have no `MapView` in `sdk-base`, it's located in `sdk`.